frontend/src/js/preview-v2/HeadlineStats.tsx

```diff
@@ -22,30 +30,56 @@ export type HeadlineStatsProps = {
   statistics: PreviewStatisticsResponse | null;
 };
 
-export default function HeadlineStats({ statistics }: HeadlineStatsProps) {
-  const { total, dateRange, statistics: statisticsList } = statistics || {};
-  const missingValues = statisticsList?.reduce(
-    (acc, obj) => acc + (obj?.nullValues ?? 0),
-    0,
-  );
+export default function HeadlineStats(
+  {statistics}: HeadlineStatsProps,
+) {
+  if (statistics === null) {
+    return (
+      <Root>
+        <Key>Zeilen:</Key>
+        <Key>Min Datum:</Key>
+        <Key>Max Datum:</Key>
+        <Key>Datumsbereich:</Key>
+        <Key>Fehlende Werte:</Key>
+      </Root>
+    );
```

Comment:
I think that's unnecessary code duplication.
Did you remove the conditional chaining solution for a reason?